client/src/components/navbar/NavBar.css

```diff
@@ -0,0 +1,30 @@
+.top-navbar {
+    width: 100%;
+    height: 11%;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 0;
+    margin: 0;
+    border-bottom: 3px solid #CECECE;
+}
+
+.leftNavbar {
+    display: flex;
+    justify-content: flex-start;
+    margin-left: 1.5%;
+}
+
+.rightNavbar  {
+    display: flex;
+
+}
```

Comment:
You should add some space between your elements in the right navbar by giving the individual links padding